src/lib/utilities/decode-local-activity.ts

```diff
@@ -71,7 +74,8 @@ export const decodeLocalActivity = async (
       convertedAttributes,
     ) as DecodedLocalActivity;
 
-    const payload = decodedAttributes?.details?.data?.payloads?.[0];
+    const payload = (decodedAttributes?.details?.data?.payloads ||
+      decodedAttributes?.details?.type?.payloads)?.[0];
```

Comment:
Would this ever hit the `decodedAttributes?.details?.type?.payloads)?.[0]` value? In order for it access `[0]`, `decodedAttributes?.details?.data?.payloads` would have to exist anyways.

Reply:
```
(decodedAttributes?.details?.data?.payloads ||
      decodedAttributes?.details?.type?.payloads)
```

Reply:
So it should hit decodedAttributes?.details?.type?.payloads if theres no decodedAttributes?.details?.data?.payloads, then access the first element

Reply:
Ah got it 👍

Reply:
No I updated it to:

```
export type DecodedLocalActivity = {
  details?: {
    data?: {
      payloads?: Payload[];
    };
    type?: {
      payloads?: Payload[];
    };
  };
};
```

Reply:
Wow I totally didn't pay attention to where the `()` were in that statement. Please disregard 🤦‍♀️

Reply:
to be fair this is an abomination to the eyeballs

```
(decodedAttributes?.details?.data?.payloads ||
      decodedAttributes?.details?.type?.payloads)?.[0];
``